### PR TITLE
Mobile fixes

### PR DESCRIPTION
**Pull request by wsvincent**
Fix overflow subheadline, bigger images, also headline text to keep/not keep.

### CODE REVIEW

File: layouts/press/single.html [hunk 1]
```diff
@@ -103,7 +103,7 @@ <h1 style="padding-bottom:10px;"> Press Resources</h1>
                 <div class="col-md-6">
                     <h3>Tagline</h3>
                     <p class="jobtext">
-                        Google for your files. Search your Desktop, Google Drive, Dropbox, Gmail, Evernote, and everything else the way you’d search the Internet.
+                        Google for your files. Search your Desktop, Google Drive, Dropbox, Gmail, Slack, Evernote, Trello, and everything else the way you’d search the Internet.
```
Comment (jasonbriggs) [inline layouts/press/single.html]: Let's keep the original for now. I agree that we need those logos closer to the top though.

**Comment by jasonbriggs** [comment]
Great responsive fixes! I think we should hold off on the copy changes for now, although we should definitely get all the services up there soon.

**Comment by wsvincent** [comment]
Sounds good.

**Comment by jasonbriggs** [comment]
Do you want to set this copy back or should I? Either way works for me.

**Comment by wsvincent** [comment]
I'll do it

**Comment by jasonbriggs** [comment]
Nice work! 👍

**Pull request merged**